index.bs

```diff
@@ -3783,6 +3915,9 @@ argument doesn't affect function's <code>this</code> binding.
 
 Issue: TODO: Add timeout argument as described in the script.evaluate.
 
+Issue: TODO: Add ownership model parameter.
```

Comment:
Why don't we just fix this? I'm not a fan of leaving a TODO to actually expose the feature we've got the backend implementation for, and it's just a matter of adding a single optional parameter to the CDDL and a single step to the algorithm.

Reply:
done

Reply:
As [discussed](https://github.com/w3c/webdriver-bidi/issues/90#issuecomment-1110841620), I made `none` a default to prevent potential memory leaks.